pkgs/by-name/ai/air-formatter/package.nix

```diff
@@ -7,19 +7,19 @@
 }:
 rustPlatform.buildRustPackage (finalAttrs: {
   pname = "air-formatter";
-  version = "0.7.0";
+  version = "0.7.1";
 
   src = fetchFromGitHub {
     owner = "posit-dev";
     repo = "air";
     tag = finalAttrs.version;
-    hash = "sha256-jFOhTol5e3TcU217HgcCKutydTePmc5/viilgiJqpqE=";
+    hash = "sha256-FGucu5emmBtxVloun7Kh+MWKZDYdKw+75L7g3GqQ7Tw=";
   };
 
   # Remove duplicate entries from cargo lock
-  cargoPatches = [ ./cargo-lock.patch ];
+  # cargoPatches = [ ./cargo-lock.patch ];
```

Comment:
Should we remove this part now that the file has been deleted?

Reply:
Definitely. Done!